conda/recipes/rmm/meta.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# Usage:
+#   conda build -c nvidia -c rapidsai -c conda-forge .
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set git_revision_count=environ.get('GIT_DESCRIBE_NUMBER', 0) %}
+package:
+  name: rmm
+  version: {{ version }}
+
+source:
+  path: ../../..
+
+build:
+  number: {{ git_revision_count }}
+  script_env:
+    - RMM_BUILD_NO_GPU_TEST
+
+requirements:
+  build:
+    - cmake >=3.12.4
+  host:
+    - python
+    - librmm {{ version }}
+    - numba >=0.41
+    - setuptools
+    - cffi
```

Comment:
Are `numba` and `cffi` needed for the build? If not, it would be good to drop them to keep `host` minimal.

Reply:
CFFI is needed for the build (for now) as the bindings are generated. As far as I can tell Numba isn't needed for the build so will try to move that out.